packages/@lwc/node-reactions/src/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import './global/init';
```

Comment:
IMO, it would make sense to consolidate the `index.ts`, `api.ts` and `init.ts` in a single `index.ts` file. It feels wrong that the `api.ts` imports `init.ts`.

Reply:
init.ts has the initialization routine. So didn't want that to end up in the main file of the module.
api.ts is basically a redirection with some dev mode asserts. Also didn't want that to be the first thing somebody sees in index.ts. I could rename it if you suggest a better name.
index.ts is really the entry point into the library.

Reply:
The strange part to me is that `api.ts` imports `init.ts`. If anything the init file should put the library in a consumable state but should expose properties that other modules consume.